src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/health/HealthInfo.kt

```diff
@@ -1,16 +0,0 @@
-package uk.gov.justice.digital.hmpps.hmppstemplatepackagename.health
-
-import org.springframework.boot.actuate.health.Health
-import org.springframework.boot.actuate.health.HealthIndicator
-import org.springframework.boot.info.BuildProperties
-import org.springframework.stereotype.Component
-
-/**
- * Adds version data to the /health endpoint. This is called by the UI to display API details
- */
-@Component
-class HealthInfo(buildProperties: BuildProperties) : HealthIndicator {
-  private val version: String = buildProperties.version
-
-  override fun health(): Health = Health.up().withDetail("version", version).build()
```

Comment:
`version` is on the `/info` endpoint so no need to be on this endpoint too.  It isn't really part of health.